ratis-docs/content/cli.md

```diff
@@ -23,6 +23,37 @@ Ratis-shell is the command line interface of Ratis.
 > Ratis-shell is currently only **experimental**.
 > The compatibility story is not considered for the time being.
 
+
+## Get the ratis-shell source 
```

Comment:
Since this section is quite short.  Let's call it "Setting up the ratis-shell" for including the entire setup and don't have any subsections.

Reply:
done